wallet/src/de/schildbach/wallet/ui/staking/StakingActivity.kt

```diff
@@ -96,7 +97,14 @@ class StakingActivity : LockScreenActivity() {
         when (status) {
             OnlineAccountStatus.None -> { }
             OnlineAccountStatus.Linking, OnlineAccountStatus.SigningUp -> super.turnOffAutoLogout()
-            else -> super.turnOnAutoLogout()
+            OnlineAccountStatus.Validating -> super.turnOffAutoLogout()
+            else -> {
+                val isWebView = navController.currentDestination?.id == R.id.crowdNodeWebViewFragment
+                if (isWebView) {
+                    navController.navigate(WebViewFragmentDirections.webViewToPortal())
+                }
+                super.turnOnAutoLogout()
+            }
```

Comment:
Not sure if this works.  Code was adapted from this change:

https://github.com/dashpay/dash-wallet/commit/a7f5073e5a4fb4a5c62f00037c5dbcd09ead284d#diff-1a8247bf710f1f2c58895df41fea7d3b06c50ca531744bda31260ee654671398L96-L101

Reply:
This is not going to work. We need to move from the `WebView` to the portal when the status is changed from `linking` to `validating`, so the logic should be in the `OnlineAccountStatus.Validating` case.

This will also avoid issues with another place where this WebView is used which is creating the new account. That flow doesn't have `.linking` or `.validating` cases.